src/UtilWidgets/include/UtilWidgets/NoticeWidget.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef UTIL_WIDGETS_NOTICE_WIDGET_H_
+#define UTIL_WIDGETS_NOTICE_WIDGET_H_
+
+#include <QWidget>
```

Comment:
Can you also include <string>, <memory> and <QPaintEvent> to make this header self contained?

Reply:
Done.